dll/win32/shell32/wine/pidl.c

```diff
@@ -1087,6 +1087,14 @@ static HRESULT _ILParsePathW(LPCWSTR path, LPWIN32_FIND_DATAW lpFindFile,
     return ret;
 }
 
+LPITEMIDLIST SHELL32_CreateSimpleIDListFromPath(LPCWSTR pszPath, DWORD dwAttributes)
```

Comment:
Put it inside n `#ifdef __REACTOS__` guard? unless the file isn't synced anymore?

Reply:
I think we have decided not to sync shell32.